tests/acceptance/visual-regression-test.js

```diff
@@ -18,34 +18,31 @@ module('Acceptance | visual regression', function(hooks) {
       { route: '/legal/', title: '/legal.html' },
       { route: '/', title: '/index.html' },
       { route: '/mascots/', title: '/mascots.html' },
-      { route: '/meetup-assets/', title: '/meetup-assets.html' },
       { route: '/learn/', title: '/learn.html' },
       { route: '/guidelines/', title: '/guidelines.html' },
       { route: '/logos/', title: '/logos.html' },
       { route: '/sponsors/', title: '/sponsors.html' },
       { route: '/ember-users/', title: '/ember-users.html' },
       { route: '/community/meetups/', title: '/community/meetups.html' },
       { route: '/community/', title: '/community/index.html' },
-      { route: '/community/meetups-getting-started/', title: '/community/meetups-getting-started.html' },
+      {
+        route: '/community/meetups/assets',
+        title: '/meetup-assets.html'
+      },
```

Comment:
can you please change these to be one-line like the rest of the things on this page?